build.sbt

```diff
@@ -97,12 +97,26 @@ lazy val vision = project.dependsOn(coreJVM).settings(
   libraryDependencies ++= jvmDependencies
 )
 
-lazy val frc = project.dependsOn(coreJVM, sensors).settings(
+lazy val frc = crossProject(JVMPlatform, NativePlatform).crossType(CrossType.Pure).dependsOn(core, sensors).settings(
   name := "potassium-frc",
-  libraryDependencies ++= sharedDependencies.value,
-  libraryDependencies ++= jvmDependencies
+  libraryDependencies ++= sharedDependencies.value
+).jvmSettings(
+  libraryDependencies ++= jvmDependencies,
+  resolvers += "WPILib-Maven" at "http://team846.github.io/wpilib-maven",
+
+  libraryDependencies += "edu.wpi.first" % "wpilib" % "2018.1.1",
+  libraryDependencies += "edu.wpi.first" % "ntcore" % "2018.1.1",
+  libraryDependencies += "com.ctre" % "phoenix" % "5.1.3.1"
+).nativeSettings(
+  resolvers += "Funky-Repo" at "http://team846.github.io/repo",
+  libraryDependencies += "com.lynbrookrobotics" %%% "wpilib-scala-native" % "0.1-SNAPSHOT",
+//  libraryDependencies += "com.lynbrookrobotics" % "ntcore" % "2018.1.1"
```

Comment:
Is this a mistake?

Reply:
Just a reminder that at some point I actually need to make networktables run on Scala Native.

Reply:
Oh, I see, this is in `nativeSettings`